### PR TITLE
pr practice

### PR DESCRIPTION
**Pull request by Yexiansen66**
Should I pull a request like this？

### CODE REVIEW

File: firstpr/fusen.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+I am Fusen Ye from China.
+A beginner of HTML,CSS and JavaScript.
+I want to contribute to the project.
+This my fisrt pr.
```
Comment (arthurbuhl) [inline firstpr/fusen.md:4]: Hi Fusen, congrats on your first PR :tada: 

You just have a small typo in this line. Its `first` actually not `fisrt`.

Thanks!

Comment (Yexiansen66) [reply firstpr/fusen.md:4]: Thank you!
I will be careful next time.

Comment (proful) [reply firstpr/fusen.md]: Please fix the typo as suggested by Arthur.

Post that we can merge

Comment (Yexiansen66) [reply firstpr/fusen.md]: I have fixed the mistake.What should I do?I can not pr agiain.

Comment (Yexiansen66) [reply firstpr/fusen.md]: @proful thank you！

**Comment by proful** [comment]
@arthurbuhl can you confirm all good?

**Comment by arthurbuhl** [comment]
Yes, this can be merged now :+1:

**Pull request merged**